tests/test_pe.py

```diff
@@ -554,5 +554,96 @@ def hook_stop_address(ql):
             ql.amsint32_driver.run(0x102D0)
 
 
+    def test_pe_win_x86_driver(self):
```

Comment:
Would it be better to add this in a new file, for example: `test_pe_driver.py` ?

Reply:
here should be fine. since unless we have more.

again if we wanna move (which is ok), we need to move linux and mac too. just to be in sync.